<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMsg</class>
 <widget class="QMainWindow" name="DialogMsg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор сообщения</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="TableMailAttr" name="tableViewAttr">
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayoutEditMsg">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutSubj">
          <item>
           <widget class="QLabel" name="labelSubj">
            <property name="text">
             <string>Тема</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSubj"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="TextEditWidget" name="textEditMsg"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionSend"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionPrintPreview"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExportPDF"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Инструменты</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSend"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
   <addaction name="actionExportPDF"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPrintPreview">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/fileprint.png</normaloff>:/icon/icons/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Просмотр печати</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/fileprint.png</normaloff>:/icon/icons/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Печать...</string>
   </property>
  </action>
  <action name="actionExportPDF">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/exportpdf.png</normaloff>:/icon/icons/exportpdf.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт в PDF...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/exit.png</normaloff>:/icon/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSend">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/send.png</normaloff>:/icon/icons/send.png</iconset>
   </property>
   <property name="text">
    <string>Отправить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/filesave.png</normaloff>:/icon/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/editundo.png</normaloff>:/icon/icons/editundo.png</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/editredo.png</normaloff>:/icon/icons/editredo.png</iconset>
   </property>
   <property name="text">
    <string>Вернуть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/editcut.png</normaloff>:/icon/icons/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/editcopy.png</normaloff>:/icon/icons/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/icons/editpaste.png</normaloff>:/icon/icons/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Выделить все</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEditWidget</class>
   <extends>QTableView</extends>
   <header>texteditwidget.h</header>
  </customwidget>
  <customwidget>
   <class>TableMailAttr</class>
   <extends>QTableView</extends>
   <header>tablemailattr.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>activated()</signal>
   <receiver>DialogMsg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
